packages/icon-button/src/icon-button-base.ts

```diff
@@ -27,28 +27,36 @@ export class IconButtonBase extends BaseElement {
 
   @query('.mdc-icon-button') protected mdcRoot!: HTMLElement;
 
+  @query('slot[name="offIcon"]') protected offIconSlot!: HTMLSlotElement;
```

Comment:
There's a very annoying bug that means we can't use the `HTMLSlotElement` type in this position, explained here: https://github.com/material-components/material-components-web-components/blob/c8e187b2f74a62902d418b6b5963052e49e19fa0/packages/formfield/src/mwc-formfield-base.ts#L75

It means that when you compile with `emitDecoratorMetadata` (which we do internally), then this code breaks in IE/Edge.

Reply:
Done